### PR TITLE
New flag for to include/exclude cover and filter pages in Liveboard PDF

### PR DESCRIPTION
**Pull request by jishnupavithran007**

### CODE REVIEW

File: src/embed/liveboard.ts [hunk 1]
```diff
@@ -651,7 +666,7 @@ export class LiveboardEmbed extends V1Embed {
      */
     public trigger<HostEventT extends HostEvent, PayloadT>(
         messageType: HostEventT,
-        data: TriggerPayload<PayloadT, HostEventT> = ({} as any),
+        data: TriggerPayload<PayloadT, HostEventT> = {} as any,
```
Comment (yinstardev) [inline src/embed/liveboard.ts]: - rever this one.

File: src/embed/liveboard.ts [hunk 2]
```diff
@@ -518,7 +533,7 @@ export class LiveboardEmbed extends V1Embed {
         params[Param.ShowLiveboardVerifiedBadge] = showLiveboardVerifiedBadge;
         params[Param.ShowLiveboardReverifyBanner] = showLiveboardReverifyBanner;
         params[Param.HideIrrelevantFiltersInTab] = hideIrrelevantChipsInLiveboardTabs;
-
+        params[Param.CoverAndFilterOptionInPDF] = coverAndFilterOptionInPDF;
```
Comment (yinstardev) [inline src/embed/liveboard.ts]: Add it conditional only if it's false, 

If the scaligent requires flag to be present in the url no matter the value : then add it here. 

Otherwise just add it only when it's required.

File: src/types.ts [hunk 3]
```diff
@@ -4728,6 +4729,16 @@ export enum Action {
...
      */
     EditTokens = 'editTokens',
+    /**
+     * Action ID for hide checkboxes for include or exclude
+     * cover and filter pages in the Liveboard PDF
+     *  @example
+     * ```js
+     * hiddenAction: [Action.CoverAndFilterOptionInPDF]
+     * ```
+     *  @version SDK: 1.36.0 | ThoughtSpot Cloud: 10.8.0.cl
```
Comment (yinstardev) [inline src/types.ts]: Update version to : 1.38.0 | for 10.8.0.cl

File: src/embed/liveboard.spec.ts [hunk 4]
```diff
@@ -322,6 +322,21 @@ describe('Liveboard/viz embed tests', () => {
...
+        } as LiveboardViewConfig);
+        liveboardEmbed.render();
+        await executeAfterWait(() => {
+            expectUrlMatchesWithParams(
+                getIFrameSrc(),
+                `http://${thoughtSpotHost}/?embedApp=true${defaultParams}&coverAndFilterOptionInPDF=true${prefixParams}#/embed/viz/${liveboardId}`,
+            );
+        });
+    });
+
```
Comment (ruchI9897) [inline src/embed/liveboard.spec.ts:339]: Can we add test for negative scenerio

**Review by ruchI9897 — APPROVED** [review]
